src/main/java/com/spotify/docker/client/DefaultDockerClient.java

```diff
@@ -57,11 +57,7 @@
 import org.apache.http.conn.ssl.SSLConnectionSocketFactory;
 import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
 import org.glassfish.hk2.api.MultiException;
```

Comment:
MultiException is in hk2-api which doesn't seem to conflict with RESTEasy anyways so I don't think I need to worry about getting rid of the reference, although explicitly adding the maven dependency rather than using it transitively would be good. I'll make that change.

Base64 I didn't notice and I assume would've failed if I tried to use it with registry authentication. Any objections to switching that to commons-codec?